index.js

```diff
@@ -14,4 +13,29 @@ app.get('/', (req, res) => res.send("Hello"));
 function onListen() {
     console.log(`Listening on port ${port}`);
 }
-app.listen(port, onListen);
\ No newline at end of file
+app.listen(port, onListen);
+
+// connect to heroku database
+const { Client } = require('pg');
+
+const client = new Client({
+    host: 'ec2-18-233-32-61.compute-1.amazonaws.com',
+    port: 5432,
+    user: 'gadyrjddeeskwc',
+    password: '8a6d028082f3a13122125822711519bb0b547f597301b1e6006f89978c04e5aa',
+    database: 'd4i5njekim7upa',
+    ssl: {
+        rejectUnauthorized: false
+      }
+  })
+
+client.connect();
```

Comment:
Should probably connect to the DB before we start listening on the port in case we have someone spamming a request